From 5a019ed53f02f7cd66d7d0ce2511c8a85983f2a2 Mon Sep 17 00:00:00 2001
From: Howard Hsu <howard-yh.hsu@mediatek.com>
Date: Tue, 10 Jun 2025 13:38:10 +0800
Subject: [PATCH 010/119] mtk: mt76: mt7996: remove the mem_total field of
 STA_REC_BF command

Remove the updating of mem_total in STA REC BF since it is not used by
the firmware.

Signed-off-by: Howard Hsu <howard-yh.hsu@mediatek.com>
---
 mt7996/mcu.c | 13 -------------
 1 file changed, 13 deletions(-)

diff --git a/mt7996/mcu.c b/mt7996/mcu.c
index 09c9627d..ce4c2bfd 100644
--- a/mt7996/mcu.c
+++ b/mt7996/mcu.c
@@ -1812,19 +1812,6 @@ mt7996_mcu_sta_bfer_tlv(struct mt7996_dev *dev, struct sk_buff *skb,
 			bf->mem_20m = bf->nrow < BF_MAT_ORDER ?
 				      matrix[bf->nrow][bf->ncol] : 0;
 	}
-
-	switch (link_sta->bandwidth) {
-	case IEEE80211_STA_RX_BW_160:
-	case IEEE80211_STA_RX_BW_80:
-		bf->mem_total = bf->mem_20m * 2;
-		break;
-	case IEEE80211_STA_RX_BW_40:
-		bf->mem_total = bf->mem_20m;
-		break;
-	case IEEE80211_STA_RX_BW_20:
-	default:
-		break;
-	}
 }
 
 static void
-- 
2.45.2

